imports/api/chain/server/methods.js

```diff
@@ -63,6 +68,8 @@ Meteor.methods({
             }
             chain.activeVotingPower = activeVP;
 
+
+            Chain.update({chainId:chain.chainId}, {$set:chain}, {upsert: true});
```

Comment:
I think we should remove the `upsert: true` here and make sure the chain data exists as the database will only store one chain record and it will be created at genesis. Therefore, at every block, the chain should be there and if there are issues update that particular record, it will be a bug.

Reply:
Maybe this should be a separate issue.

Reply:
issue created: #162